### PR TITLE
New version of CompMusic Turkish Makam Dataset

### PR DESCRIPTION
**Pull request by genisplaja**
Addresses #551

### CODE REVIEW

File: tests/datasets/test_otmm_makam.py [hunk 1]
```diff
@@ -16,11 +16,11 @@ def test_track():
         "track_id": "cafcdeaf-e966-4ff0-84fb-f660d2b68365",
         "pitch_path": os.path.join(
             os.path.normpath("tests/resources/mir_datasets/compmusic_otmm_makam/"),
-            "MTG-otmm_makam_recognition_dataset-f14c0d0/data/Kurdilihicazkar/cafcdeaf-e966-4ff0-84fb-f660d2b68365.pitch",
+            "MTG-otmm_makam_recognition_dataset-55ce75a/data/Kurdilihicazkar/cafcdeaf-e966-4ff0-84fb-f660d2b68365.pitch",
```
Comment (guillemcortes) [inline tests/datasets/test_otmm_makam.py:19]: could you `os.path.normpath()` this too?

File: tests/datasets/test_otmm_makam.py [hunk 2]
```diff
@@ -16,11 +16,11 @@ def test_track():
...
         ),
         "mb_tags_path": os.path.join(
             os.path.normpath("tests/resources/mir_datasets/compmusic_otmm_makam/"),
-            "MTG-otmm_makam_recognition_dataset-f14c0d0/data/Kurdilihicazkar/cafcdeaf-e966-4ff0-84fb-f660d2b68365.json",
+            "MTG-otmm_makam_recognition_dataset-55ce75a/data/Kurdilihicazkar/cafcdeaf-e966-4ff0-84fb-f660d2b68365.json",
```
Comment (guillemcortes) [inline tests/datasets/test_otmm_makam.py:23]: Same as above

File: scripts/make_compmusic_otmm_makam_index.py [hunk 3]
```diff
@@ -5,12 +5,12 @@
 from mirdata.validate import md5
 
 
-OTMM_MAKAM_INDEX_PATH = '../mirdata/datasets/indexes/otmm_makam_index.json'
+OTMM_MAKAM_INDEX_PATH = '../mirdata/datasets/indexes/compmusic_otmm_makam_index_dlfm2016-fix1.json'
```
Comment (guillemcortes) [inline scripts/make_compmusic_otmm_makam_index.py:8]: Also normpath this as well?

**Review by guillemcortes — CHANGES_REQUESTED** [review]
Looks good, just a couple of minor changes :)

File: tests/datasets/test_otmm_makam.py (hunk 1 above)
Comment (genisplaja) [reply tests/datasets/test_otmm_makam.py:19]: yes of course :)

**Review by guillemcortes — APPROVED** [review]
All good! Ready to merge

**Pull request merged**